src/cxx_support_test_cpp.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Build in test mode only to test cxx integration.
+
+#include <cstdint>
+#include <memory>
+
+#pragma once
```

Comment:
I really hate pragma once :upside_down_face: 

Can you write normal include guards (and make sure they come before the includes, as with other Google C++).

Reply:
Done